Delegate/SimpleTypeError.pm

```diff
@@ -433,6 +432,11 @@ sub get_delegate_info {
         undef,
         'File names may not contain a leading dot.',
     ],
+    WEAK_PASSWORD => [
+        83,
+        undef,
+        'Password does not sufficiently protect against potential attackers. Do not use a common, easy to guess password, your user id, login name, or email address.',
```

Comment:
> Password does not sufficiently protect against potential attackers.

I read online about error messages. We may want to make this more explicit to give people a clue as to the error they got, e.g. "similar to your name" or "similar to your email". However, in the interests of getting this out the door, let's simplify the error message:

Password is similar to your name, email address, and/or common phrases. Please enter a unique password.